src/import-page/file-section/FileSection.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { Card, Dropzone } from '@edx/paragon';
+import moment from 'moment';
+
+import { RequestStatus } from '../../data/constants';
+import { IMPORT_STAGES } from '../data/constants';
+import {
+  getCurrentStage, getError, getFileName, getImportTriggered,
+} from '../data/selectors';
+import {
+  reset, updateCurrentStage, updateFileName, updateImportTriggered, updateProgress, updateSavingStatus,
+} from '../data/slice';
+import { startCourseImporting } from '../data/api';
+import { setImportCookie } from '../utils';
+import messages from './messages';
+
+const FileSection = ({ intl, courseId }) => {
+  const dispatch = useDispatch();
+  const importTriggered = useSelector(getImportTriggered);
+  const currentStage = useSelector(getCurrentStage);
+  const fileName = useSelector(getFileName);
+  const { hasError } = useSelector(getError);
+  const isShowedDropzone = !importTriggered || currentStage === IMPORT_STAGES.SUCCESS || hasError;
+
+  const handleProcessUpload = async ({ fileData, requestConfig, handleError }) => {
+    try {
+      dispatch(reset());
+      dispatch(updateSavingStatus(RequestStatus.PENDING));
+      dispatch(updateImportTriggered(true));
+      dispatch(updateFileName(fileData.get('file').name));
```

Comment:
you use `fileData.get('file')` three times, maybe it would be better to create a variable?